src/us/kbase/narrativejobservice/sdkjobs/SDKMethodRunner.java

```diff
@@ -150,24 +150,15 @@ public static String runJob(RunJobParams params, AuthToken authPart,
 		// Config switch to switch to calling new Condor Utils method submitToCondor
 		if( config.get( NarrativeJobServiceServer.CFG_PROP_CONDOR_MODE ).equals( "1" ) ) {
 
+		    //TODO REMOVE
 			System.out.println("sh submit.sh " + ujsJobId);
-			String condorID = CondorUtils.submitToCondorCLI(ujsJobId,authPart.getToken(),aweClientGroups,selfExternalUrl);
-
-			//There can be a race condition here
-			System.out.println("condorID: " + condorID);
+			//TODO MOVE TO CONFIG FILE AND MAKE SURE THAT KB_DEPLOYMENT FILE MATCHES THIS
+			selfExternalUrl = kbaseEndpoint + "/njs2";
```

Comment:
NJS2 should be fixed for deployment. Currently this is being passed as a hardcoded value here and above in the original code. Also the kb_deployment_config file has njs_wrapper instead of njs2, so even if it was a config value we still can't use it yet.